qt/widgets/instrumentview/src/XIntegrationControl.cpp

```diff
@@ -178,7 +178,9 @@ void XIntegrationScrollBar::updateMinMax() {
 XIntegrationControl::XIntegrationControl(InstrumentWidget *instrWindow)
     : QFrame(instrWindow), m_instrWindow(instrWindow), m_totalMinimum(0),
       m_totalMaximum(1), m_minimum(0), m_maximum(1) {
-  m_scrollBar = new XIntegrationScrollBar(this);
+  // if the integration control panel is hidden, most of the fields should not
```

Comment:
What do you mean should not be declared? If they should not, can you avoid declaring then?

Reply:
I just forgot a comment there ...